### PR TITLE
Add zerocopy example showing how to work with structured data without deserialization

### PR DESCRIPTION
**Pull request by spacejam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ tree.compare_and_swap(k, Some(v1), Some(v2));
 tree.flush();
 ```
 
+If you would like to work with structured data without paying expensive deserialization costs, check out the [zerocopy](examples/zerocopy.rs) example!
```
Comment (Kerollmops) [inline README.md]:
```suggestion
If you would like to work with structured data without paying expensive deserialization costs, check out the [zerocopy](examples/structured.rs) example!
```

:)

Comment (Kerollmops) [reply README.md]: Whoops, too late!

Comment (spacejam) [reply README.md]: thanks for checking :)

**Comment by spacejam** [comment]
Closes #992.

**Pull request merged**